docs/website/docs/walkthroughs/deploy-a-pipeline/deploy-with-dagster.md

```diff
@@ -0,0 +1,132 @@
+by signing up for the trial.
+
+## Building Data Pipelines with `dlt`
+
+`dlt` is an open-source Python library that allows you to declaratively load data sources into
+well-structured tables or datasets through automatic schema inference and evolution. It simplifies
+building data pipelines by providing functionality to support the entire extract and load process.
+
+How does `dlt` integrate with Dagster for pipeline orchestration?
+
+`dlt` integrates with Dagster for pipeline orchestration, providing a streamlined process for
+building, enhancing, and managing data pipelines. This enables developers to leverage `dlt`'s
+capabilities for handling data extraction and load and Dagster's orchestration features to
+efficiently manage and monitor data pipelines.
+
+Here's a concise guide to orchestrating a `dlt` pipeline with Dagster, using the project "Ingesting
+GitHub issue data from a repository and storing it in BigQuery" as an example, detailed in the
+article
+[“Orchestrating unstructured data pipelines with dagster and dlt."](https://dagster.io/blog/dagster-dlt)
+
+1. Create a `dlt` pipeline. For detailed instructions on creating a pipeline, please refer to the
+   [documentation](https://dlthub.com/docs/walkthroughs/create-a-pipeline).
+
+1. Set up a Dagster project, configure resources, and define the asset as follows:
+
+   1. To create a Dagster project:
+      ```sh
+      mkdir dagster_github_issues  
+      cd dagster_github_issues  
+      dagster project scaffold --name github-issues  
+      ```
+   1. Define `dlt` as a Dagster resource:
+      ```py
+      from dagster import ConfigurableResource  
+      from dagster import ConfigurableResource  
+      import dlt  
+
+      class DltResource(ConfigurableResource):  
+          pipeline_name: str  
+          dataset_name: str  
+          destination: str  
+
+          def create_pipeline(self, resource_data, table_name):  
+       
+              # configure the pipeline with your destination details  
+              pipeline = dlt.pipeline(  
+                  pipeline_name=self.pipeline_name, 
+                                destination=self.destination, 
+                                dataset_name=self.dataset_name  
+              )  
+
+              # run the pipeline with your parameters  
+              load_info = pipeline.run(resource_data, table_name=table_name)  
+
+              return load_info  
+      ```
+   1. Define the Asset:
+      ```py
+      @asset  
+      def issues_pipeline(pipeline: DltResource):  
+          
+          logger = get_dagster_logger()  
+          results = pipeline.create_pipeline(github_issues_resource, table_name='github_issues')  
+          logger.info(results)  
+      ```
+      > For more information, please refer to
+      > [Dagster’s documentation.](https://docs.dagster.io/getting-started/quickstart)
+
+1. Next, define Dagster definitions, start the web server, and materialize the asset.
```

Comment:
Sorry @dat-a-man. I forgot to add the definitions:

```
all_assets = load_assets_from_modules([assets])
simple_pipeline = define_asset_job(name="simple_pipeline", selection= ['issues_pipeline'])

defs = Definitions(
    assets=all_assets,
    jobs=[simple_pipeline],
    resources={
        "pipeline": DltResource(
            pipeline_name = "github_issues",
            dataset_name = "dagster_github_issues",
            destination = "bigquery",
            table_name= "github_issues"
        ),
    }
)
```

Reply:
Added